vertx-mongo-service/src/main/java/io/vertx/ext/mongo/MongoService.java

```diff
@@ -99,16 +101,20 @@ MongoService bulkWriteWithOptions(String collection, List<BulkOperation> operati
   MongoService find(String collection, JsonObject query, Handler<AsyncResult<List<JsonObject>>> resultHandler);
 
   @Override
-  @Fluent
-  MongoService findBatch(String collection, JsonObject query, Handler<AsyncResult<JsonObject>> resultHandler);
+  @GenIgnore
+  default ReadStream<JsonObject> findBatch(String collection, JsonObject query) {
```

Comment:
What's the reason behind this "stub" implementation?

Reply:
The generated `MongoServiceVertxEBProxy` will not have these methods (`@GenIgnore`) but still implements `MongoService`. So we need a default impl.